core/src/main/java/org/elasticsearch/action/support/replication/TransportReplicationAction.java

```diff
@@ -550,6 +562,12 @@ public void onFailure(Exception e) {
         }
     }
 
+    /** resolves a shardId to an IndexShard. protected for overriding in tests **/
```

Comment:
This comment ("`protected`") is not consistent with the method access modifier `private`.